### PR TITLE
Executable comments

### PR DESCRIPTION
**Pull request by mayonnaise-is-an-instrument**
Added executable comments to a few Boolean methods.

### CODE REVIEW

**Comment by svenvc** [comment]
the returns (^) in the blocks are not necessary

**Comment by jgfoster** [comment]
This has a couple changes that overlap #4983 .

File: src/Kernel/Boolean.class.st [hunk 1]
```diff
@@ -95,13 +109,19 @@ Boolean >> ifFalse: alternativeBlock [
 	the argument, alternativeBlock. Create an error notification if the 
 	receiver is nonBoolean. Execution does not actually reach here because 
 	the expression is compiled in-line."
+	
+	"false ifFalse: [^ 'This statement is false!'] >>> 'This statement is false!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"false ifFalse: [ 'This statement is false!' ] >>> 'This statement is false!'" 
```

File: src/Kernel/Boolean.class.st [hunk 2]
```diff
@@ -95,13 +109,19 @@ Boolean >> ifFalse: alternativeBlock [
 	the argument, alternativeBlock. Create an error notification if the 
 	receiver is nonBoolean. Execution does not actually reach here because 
 	the expression is compiled in-line."
+	
+	"false ifFalse: [^ 'This statement is false!'] >>> 'This statement is false!'" 
+	"true ifFalse: [^ 'This statement is false!'] >>> nil"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"true ifFalse: [ 'This statement is false!' ] >>> nil"
```

File: src/Kernel/Boolean.class.st [hunk 3]
```diff
@@ -95,13 +109,19 @@ Boolean >> ifFalse: alternativeBlock [
...
 
 	self subclassResponsibility
 ]
 
 { #category : #controlling }
 Boolean >> ifFalse: falseAlternativeBlock ifTrue: trueAlternativeBlock [ 
 	"Same as ifTrue:ifFalse:."
+	
+	"true ifFalse: [ ^'That is false!' ] ifTrue: [ ^'That is true!' ]  >>> 'That is true!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"true ifFalse: [ 'That is false!' ] ifTrue: [ 'That is true!' ]  >>> 'That is true!'"
```

File: src/Kernel/Boolean.class.st [hunk 4]
```diff
@@ -95,13 +109,19 @@ Boolean >> ifFalse: alternativeBlock [
...
+	"false ifFalse: [ ^'That is false!' ] ifTrue: [ ^'That is true!' ]  >>> 'That is false!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"false ifFalse: [ 'That is false!' ] ifTrue: [ 'That is true!' ]  >>> 'That is false!'"
```

File: src/Kernel/Boolean.class.st [hunk 5]
```diff
@@ -113,6 +133,9 @@ Boolean >> ifTrue: alternativeBlock [
 	the argument, alternativeBlock. Create an error notification if the 
 	receiver is nonBoolean. Execution does not actually reach here because 
 	the expression is compiled in-line."
+	
+	"true ifTrue: [^ 'This statement is true!'] >>> 'This statement is true!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"true ifTrue: [ 'This statement is true!' ] >>> 'This statement is true!'" 
```

File: src/Kernel/Boolean.class.st [hunk 6]
```diff
@@ -113,6 +133,9 @@ Boolean >> ifTrue: alternativeBlock [
 	the argument, alternativeBlock. Create an error notification if the 
 	receiver is nonBoolean. Execution does not actually reach here because 
 	the expression is compiled in-line."
+	
+	"true ifTrue: [^ 'This statement is true!'] >>> 'This statement is true!'" 
+	"false ifTrue: [^ 'This statement is true!'] >>> nil"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"false ifTrue: [ 'This statement is true!' ] >>> nil"
```

File: src/Kernel/Boolean.class.st [hunk 7]
```diff
@@ -124,6 +147,9 @@ Boolean >> ifTrue: trueAlternativeBlock ifFalse: falseAlternativeBlock [
 	result of evaluating the argument falseAlternativeBlock. If the receiver 
 	is a nonBoolean then create an error notification. Execution does not 
 	actually reach here because the expression is compiled in-line."
+	
+	"true ifTrue: [ ^'That is true!' ] ifFalse: [ ^'That is false!' ]  >>> 'That is true!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"true ifTrue: [ 'That is true!' ] ifFalse: [ 'That is false!' ]  >>> 'That is true!'"
```

File: src/Kernel/Boolean.class.st [hunk 8]
```diff
@@ -124,6 +147,9 @@ Boolean >> ifTrue: trueAlternativeBlock ifFalse: falseAlternativeBlock [
...
+	"false ifTrue: [ ^'That is true!' ] ifFalse: [ ^'That is false!' ]  >>> 'That is false!'"
```
Comment (jecisc) [inline src/Kernel/Boolean.class.st]:
```suggestion
	"false ifTrue: [ 'That is true!' ] ifFalse: [ 'That is false!' ]  >>> 'That is false!'"
```

**Review by jecisc — CHANGES_REQUESTED** [review]
Thank you!

I did suggestions to remove the `^` as Sven said. 
In the github interface, in the "File" tab you can batch accept the suggestions and commit them.

**Comment by jgfoster** [comment]
@jecisc, Thank you for making the suggestions. It is nice for newcomers to see the community collaboration and support!

**Review by mayonnaise-is-an-instrument — COMMENTED** [review]
commited the suggestions.

**Comment by jgfoster** [comment]
Looks good to me.

**Pull request merged**